app/COVIDSafePathsConfig.ts

```diff
@@ -1,3 +1,6 @@
 import Config from 'react-native-config';
+import { AUTHORITY_ADVICE_URL } from './constants/authorities';
```

Comment:
Heads up: I recently moved AUTHORITY_ADVICE_URL to the env.bt file and merged that PR in

Reply:
Thanks for the heads up! Noticed it when I pulled in the develop branch. I fixed the merge conflicts, should be all good now!

Reply:
Not seeing those changes pushed yet, fyi